tsl/src/bgw_policy/job.c

```diff
@@ -661,3 +661,26 @@ job_execute(BgwJob *job)
 
 	return true;
 }
+
+/*
+ * Check configuration for a job type.
+ */
+void
+job_config_check(Name proc_schema, Name proc_name, Jsonb *config)
```

Comment:
I moved because the job_api.c module include the job.h and to do what you proposed I should include job_api.h into job.c, so to me it seems weird (cyclic dependency).

Reply:
Fair enough, but that suggests that something is wrong with the module structure. Nothing to fix in this pull request though.